src/ControllerStatusIO.c

```diff
@@ -114,23 +162,35 @@ BOOL Ros_Controller_Initialize()
     }
 
     //get the robot calibration data for multi-robot systems
-    for (i = 0; i < MAX_ROBOT_CALIBRATION_FILES; i += 1)
+    const BOOL bCalibLoadedOk = Ros_Controller_LoadGroupCalibrationData(&g_Ros_Controller);
+
+    //It's OK if calibration did not load (for single-group systems fi, or systems
+    //that haven't been calibrated), but post an alarm just in case IFF:
+    // - this is a multi-group system
+    // - TF broadcast is enabled
+    // - none of the calibration files loaded, and
+    // - the warning has not been disabled
+    const BOOL bShouldWarnAboutCalib =
+        g_Ros_Controller.numGroup > 1 &&
```

Comment:
This needs more intelligence to not count base tracks. I'm not sure the best way to do this, short of brute force. Just loop through each group and look at the `groupId`?

Reply:
Is it just base tracks/groups, or do we need to check for a specific combination of ids / relation between ids?

But we could easily add a new function which does this. Intended use would be during initialisation, so performance isn't too much of a concern directly (and even then: `O(n)` for small lists would not be problematic).

Reply:
@ted-miller: just to clarify: would you suppress the alarm if there are *any* base groups used for tracks? Or only in other specific cases?

Reply:
> Is it just base tracks/groups, or do we need to check for a specific combination of ids / relation between ids?

AFAICT, just the base axes.

> would you suppress the alarm if there are any base groups used for tracks? Or only in other specific cases?

Specific cases.
R1+B1: No alarm
R1+B1+S1: Alarm
R1+B1+R2: Alarm

Reply:
Would it correct to say that any multi-group system with non-base-axis groups would fall into the `raise-alarm-if-no-calibration-files-loaded` category?

IIUC, base axes groups are always associated with robot groups, correct? So that leaves just station axes and proper robots.

Reply:
That sounds correct